src/multipart.ts

```diff
@@ -100,6 +103,7 @@ export default async function* parseMultipartBody(
         }
       }
       if (buf[ptr] !== boundary.charCodeAt(i)) {
```

Comment:
this fails right now because buffer is initialized, but the contents are undefined